### PR TITLE
Add a note that Desktop isn't available for Linux

### LINKED ISSUE #6346: The download links for GitHub destktop is not clear.
### What article on docs.github.com is affected?

[GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-github-desktop)

### What part(s) of the article would you like to see updated?

The download option is not showing anything when touching it. it goes to nowhere.

### Additional information

I got to that link by typing `GitHub desktop linux` on google. 
I know there is no linux version yet, but the link that I Have provided is not clearly specifying it and the link to download option is unvalid.

### PR DESCRIPTION
**Pull request by ajsirota**
### Why:

Close #6346 by adding a note that Desktop is not available for Linux, linking to the tracking issue. (desktop/desktop#1525)

### What's being changed:

Added a note:
A screenshot of the documentation page in staging showing a note explaining that GitHub Desktop is not available for Linux.
This shows on both the Windows and Mac tabs. I did not add a Linux tab, because I thought it would be confusing to make it look like there is a way to install on Linux.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### CODE REVIEW

**Comment by janiceilene** [comment]
@ajs256 Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by A751-acm — APPROVED** [review]

**Review by A751-acm — DISMISSED** [review]
Eu preciso saber porque não fui aprovada ainda com cartão de crédito

**Comment by ajs256** [comment]
Machine translation from Portuguese of what @A751-acm said:
> I need to know why I haven't been approved with a credit card yet

...i'm not sure how I can help you with that.

This is a machine translation:

Esta é uma tradução automática:
Não tenho certeza de como posso ajudá-lo.

**Review by nguyentung-199 — DISMISSED** [review]

File: content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/installing-github-desktop.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ versions:
 
 You can install {% data variables.product.prodname_desktop %} on supported operating systems. If you have an account on {% data variables.product.prodname_dotcom %} or {% data variables.product.prodname_enterprise %}, you can connect your account to {% data variables.product.prodname_desktop %}. For more information about creating an account, see "[Signing up for a new {% data variables.product.prodname_dotcom %} account](/articles/signing-up-for-a-new-github-account/)" or contact your {% data variables.product.prodname_enterprise %} site administrator.
 
+{% note %}
+
+**Note:** GitHub Desktop is not currently available for Linux. You can subscribe to [this issue](https://github.com/desktop/desktop/issues/1525) to know when support is added.
+
+{% endnote %}
+
```
Comment (ethanpalm) [inline content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/installing-github-desktop.md:20]:
```suggestion
```

File: content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/installing-github-desktop.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ versions:
 
 You can install {% data variables.product.prodname_desktop %} on supported operating systems. If you have an account on {% data variables.product.prodname_dotcom %} or {% data variables.product.prodname_enterprise %}, you can connect your account to {% data variables.product.prodname_desktop %}. For more information about creating an account, see "[Signing up for a new {% data variables.product.prodname_dotcom %} account](/articles/signing-up-for-a-new-github-account/)" or contact your {% data variables.product.prodname_enterprise %} site administrator.
```
Comment (ethanpalm) [inline content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/installing-github-desktop.md:13]:
```suggestion
You can install {% data variables.product.prodname_desktop %} on supported operating systems, which currently include {% data variables.desktop.mac-osx-versions %} and {% data variables.desktop.windows-versions %}. If you have an account on {% data variables.product.prodname_dotcom %} or {% data variables.product.prodname_enterprise %}, you can connect your account to {% data variables.product.prodname_desktop %}. For more information about creating an account, see "[Signing up for a new {% data variables.product.prodname_dotcom %} account](/articles/signing-up-for-a-new-github-account/)" or contact your {% data variables.product.prodname_enterprise %} site administrator.
```

**Review by ethanpalm — DISMISSED** [review]
👋🏼  Hi @ajs256, thanks for opening this PR! We can definitely make it a bit more clear which operating systems are supported by GitHub Desktop. In general, we don't link to issues in the GitHub docs, so I edited that out. I also moved the info about supported operating systems to the very beginning of the article so that people can know right away if Desktop will work on their OS.

Thanks for helping improve the GitHub docs! The changes should be live within 24 hours 🎉

**Review by ethanpalm — APPROVED** [review]

**Pull request merged**